kuma/wiki/tests/test_forms.py

```diff
@@ -179,51 +181,51 @@ def test_case_sensitive_tags(self, mock_requests):
         self.assertTrue(rev_form.is_valid())
         self.assertEqual(rev_form.cleaned_data['tags'], '"JavaScript"')
 
-    @pytest.mark.spam
-    @requests_mock.mock()
-    def test_akismet_enabled(self, mock_requests):
+    def _setup_akismet_post(
```

Comment:
I don't know why this method has an underscore in front of it, who are you hiding it from?